app/pyproject.toml

```diff
@@ -4,49 +4,66 @@ version = "0.1.0"
 description = "A template flask API for building ontop of"
 packages = [{ include = "api" }]
 authors = ["Nava Engineering <engineering@navapbc.com>"]
+include = ["openapi.yml"]
 
+# Base packages for all versions of the images
+# Any packages needed for scripts called from outside of the app
+# would go here, ie create-user-csv
 [tool.poetry.dependencies]
 python = "^3.10"
-SQLAlchemy = {extras = ["mypy"], version = "^1.4.40"}
-alembic = "^1.8.1"
-psycopg2-binary = "^2.9.3"
-python-dotenv = "^0.20.0"
-pydantic = "^1.10.0"
-botocore = "^1.27.67"
-boto3 = "~1.24.67"
-smart-open = "^6.1.0"
-pytz = "^2022.2.1"
-APIFlask = "^1.1.3"
-marshmallow-dataclass = {extras = ["enum", "union"], version = "^8.5.8"}
-marshmallow = "^3.18.0"
-
-[tool.poetry.dev-dependencies]
-pytest = "^6.0.0"
-black = "^22.6.0"
-flake8 = "^5.0.4"
-flake8-bugbear = "^22.8.23"
-flake8-alfred = "^1.1.1"
-isort = "^5.10.1"
-mypy = "^0.971"
-moto = {extras = ["s3"], version = "^4.0.2"}
-types-pytz = "^2022.2.1"
-coverage = "^6.4.4"
-Faker = "^14.2.0"
-factory-boy = "^3.2.1"
-bandit = "^1.7.4"
 
+[tool.poetry.group.app]
+optional = true
+
+# Packages needed for the app, will be in the release image
+# Pin the version to a specific version to speed up build times
+## https://github.com/python-poetry/poetry/issues/2094
+[tool.poetry.group.app.dependencies]
+SQLAlchemy = {extras = ["mypy"], version = "1.4.46"}
+alembic = "1.9.1"
+psycopg2-binary = "2.9.5"
+python-dotenv = "0.21.0"
+pydantic = "1.10.4"
+botocore = "1.29.49"
+boto3 = "1.26.49"
+smart-open = "6.3.0"
+pytz = "2022.7"
+APIFlask = "1.2.1"
+marshmallow-dataclass = {extras = ["enum", "union"], version = "8.5.11"}
+marshmallow = "3.19.0"
+
+[tool.poetry.group.dev]
+optional = true
+
+# Local only packages, any packages needed for the testing suite
+# Pin the version to a specific version to speed up build times
+## https://github.com/python-poetry/poetry/issues/2094
 [tool.poetry.group.dev.dependencies]
+pytest = "7.2.0"
+black = "22.12.0"
+flake8 = "6.0.0"
+flake8-bugbear = "22.12.6"
+flake8-alfred = "1.1.1"
+isort = "5.11.4"
+mypy = "0.991"
+moto = {extras = ["s3"], version = "4.1.0"}
+types-pytz = "2022.7.0.0"
+coverage = "7.0.5"
+Faker = "16.4.0"
+factory-boy = "3.2.1"
+bandit = "1.7.4"
 pytest-watch = "^4.2.0"
 
 [build-system]
 requires = ["poetry-core>=1.0.0"]
 build-backend = "poetry.core.masonry.api"
 
 [tool.poetry.scripts]
+run-api = "api.__main__:main"
 db-migrate-up = "api.db.migrations.run:up"
 db-migrate-down = "api.db.migrations.run:down"
 db-migrate-down-all = "api.db.migrations.run:downall"
-create-user-csv = "api.scripts.create_user_csv:main"
+create-user-csv = "api.services.users.create_user_csv:main"
```

Comment:
just sharing for awareness that i have a PR that intends to change this, so it's probably ok to leave this kind of broken for now https://github.com/navapbc/template-application-flask/pull/138

Reply:
Gotcha, reverted in next commit